src/components/1-atoms/SelectStyles.js

```diff
@@ -5,11 +5,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import makeStyles from '@mui/styles/makeStyles';
-
-export const useStyles = makeStyles((theme) => ({
+export const styles = {
     form: {
-        margin: theme.spacing(1),
+        margin: 1,
         minWidth: 120,
         maxWidth: 300,
```

Comment:
in 'px' ?

Reply:
yes, it will be added automatically

Reply:
That would be clearer if we add 'px', as Thang suggested, to avoid confusion.